artifacts/farm_contract.json

```diff
@@ -1,6 +1,6 @@
 {
   "source": {
-    "hash": "0x2321d7732d8d6402e2ba8546b78ebd718c5a090c2943076dc627f3811cbaddd5",
+    "hash": "0x73dc2661c0aac78ed635172b85dd14d3357dcd54d6f64659c3fc9426b9e06990",
```

Comment:
I think we better keep the `.json` and `.contract` files as they are, because they are deployed in this version on-chain. 

The difference is probably only because of non-determinism and some versioning issues.

Reply:
Maybe we can remove and ignore these files completely because the deployed versions are in the [`deployed` directory](https://github.com/Cardinal-Cryptography/common-amm/tree/main/deployments/mainnet/contracts)?

Reply:
Sure, but please update this line iin farms script to point to the correct artifact: https://github.com/Cardinal-Cryptography/common-amm/blob/f49ac5f4b5781638ef1f528582b6cd032ae36b3b/scripts/commonfarms-cli/commonfarms-cli#L18